### PR TITLE
+act add getEventStream to ActorSystem

### PR DESCRIPTION
**Pull request by He-Pin**
refs:https://github.com/akka/akka/issues/25870

### CODE REVIEW

File: akka-actor/src/main/scala/akka/actor/ActorSystem.scala [hunk 1]
```diff
@@ -467,6 +467,11 @@ abstract class ActorSystem extends ActorRefFactory {
    */
   def eventStream: EventStream
 
+  /**
+   * Java API: Main event bus of this actor system, used for example for logging.
+   */
+  @inline def getEventStream: EventStream = eventStream
```
Comment (hepin1989) [inline akka-actor/src/main/scala/akka/actor/ActorSystem.scala]: If not sure if adding `@inline` is a good idea.

Comment (johanandren) [reply akka-actor/src/main/scala/akka/actor/ActorSystem.scala]: Yeah, skip that.

**Review by johanandren — APPROVED** [review]
LGTM, with the inline removed

**Review by chbatey — APPROVED** [review]
LGTM

**Comment by hepin1989** [comment]
Removed the `@inline`

**Pull request merged**